src/main/java/com/dynatrace/loadrunner/converter/util/BodyFilePatcherUtil.java

```diff
@@ -83,6 +83,15 @@ private static int getIndexAfterBlockComment(String commentedInstruction, int i)
 	}
 
 	public static String getFirstStringParameter(String instruction, char stringParameter) {
+		String instruction;
+        
+        // if this C language, remove comments
+        if (stringParameter == Constants.DOUBLE_QUOTE) {
+            instruction = removeComments(instructionWithComments);
+        }
+        else {
```

Comment:
follow existing formatting:
```suggestion
		} else {
```